docs/calicoctl/profile.md

```diff
@@ -0,0 +1,518 @@
+|   PROF_D   |
++------------+
+
+$ calicoctl profile show --detailed
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+|    Name    |      Host      | Orchestrator ID |                           Workload ID                            |           Endpoint ID            | State  |
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+| PROF_A_C_E | calico-host-01 |      docker     | 15a483ab65cc60e2831859dd0eb3b3acde86cc464aea8f6bbbd35cb5395ed007 | ee033eba777f11e5abe9080027b2d0eb | active |
+| PROF_A_C_E | calico-host-01 |      docker     | a549f09ed2c9ae2a840edc834dea0d14e686a4bf6195c791db283aed7f8b5e48 | 011af79a778011e5abe9080027b2d0eb | active |
+| PROF_A_C_E | calico-host-02 |      docker     | ecbb19f9913236da5e03187fbfadf65a5761c49051adf54758d7f447852336fa | f6e1dd20777f11e5abe9080027b2d0eb | active |
+|   PROF_B   | calico-host-01 |      docker     | fb028123092ed9a91f279f1aa11517cbd5b023ec4daf382cce8471b3fe41e9af | f2497552777f11e5abe9080027b2d0eb | active |
+|   PROF_D   | calico-host-02 |      docker     | 6f103f1439c3d69ed014d4aa7f5b92dac7b892cbec627ac3b181ae4263e307de | faecf8be777f11e5abe9080027b2d0eb | active |
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+
+```
+
+### calicoctl profile add <PROFILE>
+This command allows you to create security profiles to configure 
+networking policy for Calico endpoints. 
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl profile add <PROFILE>
+
+    <PROFILE>: Name of security profile to add.
+```
+
+Examples:
+
+```
+$ calicoctl profile add PROF_TEST
+Created profile PROF_TEST
+```
+
+### calicoctl profile remove <PROFILE> 
+This command allows you to remove security profiles from Calico. 
+
+If you try to remove a profile that is still associated with 
+endpoints, the command will fail and a warning message will be 
+printed.  To remove the profile anyway, you can run the command 
+with the `--no-check` flag to remove the profile without checking 
+dependencies, or you can run the [`calicoctl container`](./container.md) 
+command that removes the profile association from a container 
+for each container associated with the profile.
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl profile remove <PROFILE> [--no-check]
+
+    <PROFILE>: Name of profile to remove.
+
+    --no-check: Remove the container without checking endpoint dependencies.
+```
+
+Examples:
+
+```
+$ calicoctl profile remove PROF_TEST
+Deleted profile PROF_TEST
+```
+
+### calicoctl profile <PROFILE> tag show
+This command shows the tags related to a given profile.
+
+Tags are used to configure policy to allow/deny traffic 
```

Comment:
I'd phrase this "All endpoints with this policy profile have the tags applied.  Tags are used in rules to select which endpoints to allow or deny traffic."